src/web/command_control/client/style/CommandControl.less

```diff
@@ -1174,16 +1174,12 @@ div.closeButton {
   padding: 8pt;
   border-radius: 4pt;
 
-  button {
-    color: white;
-    background-color: #06c006;
+  #takeControlButton {
+    color: black;
```

Comment:
I wonder if this color should be a @button-text or something similar. Not sold on it, but want to make sure it's considered.